### PR TITLE
[Update] set mode dark/light for layout #12

### PR DESCRIPTION
**Pull request by dinhthi12**
# Related Tickets

- [#Ticket_ID](https://edu-redmine.vn/issues/xxxx)

## WHAT this PR does?

- ex: Change number items `completed/total` in user page.

## Notes Impacted Areas

- N/A.

## Attachments

- N/A.

### CODE REVIEW

File: client/src/layout/DefaultLayout.jsx
Comment (hoangpt1310) [inline client/src/layout/DefaultLayout.jsx:1]: add comments

File: client/src/page/Page404.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+              />
+              <CButton color="primary">Search</CButton>
+            </CInputGroup>
+
+            {/* Back to Home Button */}
+            <CRow className="justify-content-center mt-3">
+              <CButton
+                color="primary"
+                className="w-auto d-flex align-items-center"
+                onClick={() => navigate('/')} // Navigate to home
```
Comment (hoangpt1310) [inline client/src/page/Page404.jsx]: add comments

File: client/src/scss/_theme.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+body {
+  background-color: var(--cui-tertiary-bg);
+}
+
+.wrapper {
+  width: 100%;
+  @include ltr-rtl("padding-left", var(--cui-sidebar-occupy-start, 0));
+  @include ltr-rtl("padding-right", var(--cui-sidebar-occupy-end, 0));
+  will-change: auto;
+  @include transition(padding .15s);
```
Comment (hoangpt1310) [inline client/src/scss/_theme.scss]: tương tự

File: client/src/components/AppHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from "react"
+
+const AppHeader = () => {
+  return <div>App header work!</div>
+}
+
+export default React.memo(AppHeader)
```
Comment (tn0810520) [inline client/src/components/AppHeader.jsx:7]: note file knowledge.

**Pull request merged**